sim/core/spell_cast.go

```diff
@@ -467,18 +469,18 @@ func (hitEffect *SpellHitEffect) applyResistances(sim *Simulation, spellCast *Sp
 
 	if spellCast.SpellSchool.Matches(SpellSchoolPhysical) {
 		// Physical resistance (armor).
-		*damage *= 1 - hitEffect.Target.ArmorDamageReduction(spellCast.Character.stats[stats.ArmorPenetration]+hitEffect.BonusArmorPenetration)
+		*damage *= hitEffect.Target.ArmorDamageMultiplier(spellCast.Character.stats[stats.ArmorPenetration] + hitEffect.BonusArmorPenetration)
 	} else if !spellCast.SpellExtras.Matches(SpellExtrasBinary) {
 		// Magical resistance.
 		// https://royalgiraffe.github.io/resist-guide
 
```

Comment:
i got my data from parsing tens of thousands of resists as well as several other implementations. Its possible its wrong but it seemed correct. If I remember the number is actually like 12.8% or something and so I rounded to 13 instead of 12.